les/handler.go

```diff
@@ -939,7 +939,7 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 
 				var prefix string
 				if root, prefix = pm.getHelperTrie(req.Type, req.TrieIdx); root != (common.Hash{}) {
-					auxTrie, _ = trie.New(root, trie.NewDatabase(ethdb.NewTable(pm.chainDb, prefix)))
+					auxTrie, _ = trie.New(root, trie.NewDatabase(ethdb.NewTable(pm.chainDb, prefix), 1))
```

Comment:
In most cases, you just put a `0` -- why `1` specifically here?

Reply:
0 is infinite memory. In tests, I don't care. In live code however, we always want to have limits, so I set it to 1 MB here.